cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -138,6 +147,20 @@ private PublicationContext buildPublicationContext() throws InvalidIsbnException
                        .withOpenAccess(false)
                        .build();
         }
+        if (isJournal()) {
+            List<String> isbnList = new ArrayList<>();
+            isbnList.add(extractIsbn());
+            URL theUrl = new URL("https://www.example.com/");
+            return new Journal.Builder()
+                    .withLevel(HARDCODED_LEVEL)
```

Comment:
HARDCODED_JOURNAL_LEVEL

Reply:
Why cant they be re-used from the Book version?